pkg/template/master_template.go

```diff
@@ -427,7 +427,8 @@ storage:
       filesystem: root
       mode: 0644
       contents:
-        source: "data:text/plain;charset=utf-8;base64,{{  index .Files "k8s-resource/calico-crds.yaml" }}"
+        source: "https://raw.githubusercontent.com/giantswarm/k8scloudconfig/v10.2.0/files/k8s-resource/calico-crds.yaml"
```

Comment:
Additional thought: Is this guarded for KVM? What happens e.g. in aws china?

Reply:
I tested cluster creation in AWS China with the aws-operator PR linked above. The CRDs were able to be pulled from GitHub in my test cluster. I know that the rules for what is blocked there can change over time, but we don't plan to have any more aws-operator releases and it does currently work. Do you think this is safe to rely on @giantswarm/team-firecracker-engineers?

Reply:
Our helm charts are hosted on GitHub Pages which is a GitHub domain and I don't think we've had problems with that so far.

Reply:
The hardcoded value worries me a bit, I have a bad feeling that we are going to change the crds in the future and then spend a lot of time trying to figure out why they're not being applied. But not sure how we can template this though so it always gets translated to the current version.

Reply:
True. I do feel like we need to take some time to improve this setup since installing CRDs and Calico feels quite outside the scope of ignition to me. We were able to install it an app during the hive sprint, so maybe we can try that soon. I'll open an issue so we don't forget.

Reply:
> True. I do feel like we need to take some time to improve this setup since installing CRDs and Calico feels quite outside the scope of ignition to me. We were able to install it an app during the hive sprint, so maybe we can try that soon. 

I can add some of the history here. When we moved the apps manifests out of cloudconfig to helm charts we stopped at CoreDNS and didn't remove Calico because at the time we couldn't get it to bootstrap in a new cluster. 

But this was a long time ago and many things will have changed. It's good news that Calico was installable as an app during the hive sprint and yes a helm chart is probably easier to manage.

Reply:
I think we will have an issues in installations like buffalo where we need to allow each  domain we access